pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -91,60 +91,51 @@ cluster_id="42"
 
 archive=$(mktemp)
 
-if [[ "${FAILURE_POLICY}" == "fail" ]]; then
-	fail_code=1
-fi
-
-# Work around to use installer URL until we have the image.
-#if [[ "${INSTALLER_URL}" != "" ]]; then
-
 if [[ "${INSTALLER_URL}" != "" ]]; then
 	installer_url="${INSTALLER_URL}"
```

Comment:
This line above can be removed then since it's always replaced,

```
installer_url="${api_url}/v1/deployment/installer/agent/unix/paas/latest?flavor=${FLAVOR}&include=${TECHNOLOGIES}&bitness=64"
```